### DISCUSSION TITLE
DEEZER :: ERROR :: Download failed

### DISCUSSION THREAD

**Discussion by jelly-monster** [RandomNinjaAtk/docker-lidarr-extended#51] [Q&A] [answered]
Hi,
When running the audio script, I keep getting this in the logs:

> 2022-08-15 11:02:47 :: Extended Audio :: DEEZER :: deemix client setup verification...
2022-08-15 11:02:48 :: Extended Audio :: DEEZER :: ERROR :: Download failed
2022-08-15 11:02:48 :: Extended Audio :: DEEZER :: ERROR :: Please review log for errors in client
2022-08-15 11:02:48 :: Extended Audio :: DEEZER :: ERROR :: Try updating your ARL Token to possibly resolve the issue...
2022-08-15 11:02:48 :: Extended Audio :: DEEZER :: ERROR :: Exiting...

Does anyone know what could be causing this?  I've checked the ARL token, and it definately matches.

**Comment by RandomNinjaAtk** [comment, marked as answer]
You should try getting a new token and entering it, make sure there are no spaces before/after it. Tokens do expire...

**Reply by jelly-monster** [reply]
I've just tried a new token, from Firefox this time instead of Chrome.  But I get the same error as before.

**Reply by RandomNinjaAtk** [reply]
Is this album available from your geographic location/region? https://www.deezer.com/album/197472472

If it is not, that might explain why, because that is what it is using for testing...

**Reply by jelly-monster** [reply]
Yep - I can listen to it fine

**Reply by RandomNinjaAtk** [reply]
Check this file, "/config/xdg/deemix/.arl"

Make sure you see your complete token in it. If it doesn't match the token you gave the app, that is likely why, which means a docker config issue.

**Reply by jelly-monster** [reply]
This looks like the issue!  It's different.
I've tried changing it, but when the script runs it changes back.  I also removed the docker and ran the composer file again (which has the new ARL token).  But it keeps reverting back.  Is it stored anywhere else?

**Reply by RandomNinjaAtk** [reply]
You can see here:
https://github.com/RandomNinjaAtk/docker-lidarr-extended/blob/bd64615ca3501ec8a4e5ded714a1a9dcca9cceae/root/scripts/audio.sh#L746-L761

That file is generated every-time the script runs. So your issue is likely with your docker compose/run or other method your using to include the parameters and start the container. 

So its something wrong with your configuration and how your running the container... Can't really help troubleshoot it further because each setup is unique to each user from that perspective...

You could post your config for review here

**Reply by jelly-monster** [reply]
I fixed it!  I can't say exactly what I did differently from before, but it was definately the ARL token issue.  The only "big" change I made was moving the ARL token to my .env file.

Thanks for the troubleshooting help!
